.circleci/config.yml

```diff
@@ -345,6 +357,22 @@ jobs:
           command: |
             ./dev_scripts/env.py --distro debian --version bullseye build-dev
 
+      - run:
+          name: Configure Podman for Debian Bullseye
+          command: |
+            # Copy the Podman config into the container image we created for the
```

Comment:
Can we add here a reference to the issue that this extra code is trying to solve?

Reply:
I've updated the above block with a link to #388.

Reply:
can you point me to the line / commit? I can't find it